phpstan.neon.dist

```diff
@@ -10,6 +10,13 @@ parameters:
         - "~(Method|Property) .+::.+(\\(\\))? (has parameter \\$\\w+ with no|has no return|has no) typehint specified~"
         - "~Variable property access on .+~"
         - "~Variable method call on static\\(GraphQL\\\\Server\\\\ServerConfig\\)~" # TODO get rid of
+        - "~Only booleans are allowed in .*~" # TODO https://github.com/phpstan/phpstan-strict-rules/issues/2
```

Comment:
I'm working on this in small PRs just FYI (eg. https://github.com/webonyx/graphql-php/pull/503/files)